src/hotspot/share/classfile/classLoaderDataGraph.hpp

```diff
@@ -66,6 +66,7 @@ class ClassLoaderDataGraph : public AllStatic {
   static void purge(bool at_safepoint);
   static void clear_claimed_marks();
   static void clear_claimed_marks(int claim);
+  static void verify_claimed_marks_cleared(int claim);
```

Comment:
Is there a reason to not use `NOT_DEBUG_RETURN` for this method like for `ClassLoaderData::verify_not_claimed()`? I would prefer this to be consistent.

Reply:
I just think the NOT_DEBUG_RETURN / ASSERT macros pollute the code and makes it ugly to read, so I opted to not do it. This function is absolutely not performance critical, so I chose this approach.

I can be compelled to change it, if Reviewers prefer to change the code to use NOT_DEBUG_RETURN .

Reply:
I am fine with the change as is, just noting that this change uses it in one place and does not in the other, which seemed inconsistent to me.